app/src/lib/design-system/components/RQNetworkTable/index.tsx

```diff
@@ -78,10 +80,40 @@ export const RQNetworkTable: React.FC<RQNetworkTableProps> = ({
     setActiveLogId(closestLog?.log?.id);
   }, [logs, sessionCurrentOffset, sessionRecordingStartTime]);
 
+  const finalLogs = useMemo(() => {
+    let finalLogs = logs;
```

Comment:
Create a copy of logs, the above might be working now, but might cause some unexpected issues in future
```suggestion
    let finalLogs = [...logs];
```